framework/generated/generated_vulkan_struct_deep_copy_stype.cpp

```diff
@@ -34,7 +34,7 @@
 GFXRECON_BEGIN_NAMESPACE(gfxrecon)
 GFXRECON_BEGIN_NAMESPACE(graphics)
 
-inline uint8_t* offset_ptr(uint8_t* ptr, uint32_t offset)
+inline uint8_t* offset_ptr(uint8_t* ptr, uint64_t offset)
```

Comment:
size_t's size is implementation-defined, so I'd prefer something from stdint-header with known size. 
intention was to avoid triggering warnings when compiling for 32/64 systems. otherwise it's an .cpp-internal helper-function that is not exposed publicly

Reply:
so, I'm in favor of using `uint64_t` here

Reply:
Cool.  Just making sure we're being deliberate.